numerics/fixed_arrays.hpp

```diff
@@ -72,7 +72,7 @@ class FixedMatrix final {
   };
 
   template<int r>
-  Row<r> row() const;
+  typename FixedMatrix<Scalar, rows, columns>::template Row<r> row() const;
```

Comment:
You don't need the parameters on `FixedMatrix`: within the definition, `FixedMatrix` without parameters refers to the class being defined. The following should be enough:
```c++
template<int r>
typename FixedMatrix::template Row<r> row() const;

```

Reply:
Ah, didn't know that. Fixed!